source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -580,6 +580,10 @@ def __init__(self):
         if handle_poi_infos(args["add_poi_infos"], args["bss_stands"]):
             self.get_decorators.insert(1, ManageParkingPlaces(self, 'journeys'))
 
+        parser_get.add_argument(
+            "max_waiting_duration", type=PositiveInteger(), help='Maximum waiting, must be > 0'
```

Comment:
Do we really want to expose this parameter or only for debug purpose? If only for debug,  use '_max_waiting_duration' instead.

Reply:
No, this is not a debug parameter !